Firebase/Firebase/FirebaseCommunity.h

```diff
@@ -1,77 +0,0 @@
-/*
- * Copyright 2017 Google
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-#if !defined(__has_include)
-  #error "Firebase.h won't import anything if your compiler doesn't support __has_include. Please \
-          import the headers individually."
-#else
-  #if __has_include(<FirebaseCommunity/FirebaseCore.h>)
```

Comment:
What's the equivalent of this file in the new grand unified world?

Reply:
The public version of the Firebase pod - https://github.com/CocoaPods/Specs/blob/master/Specs/0/3/5/Firebase/4.6.0/Firebase.podspec.json